internal/cli/approve.go

```diff
@@ -11,36 +11,54 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var approveLabels []string
-var approveRegion string
+type ApproveOptions struct {
+	ApproveLabels []string
+	ApproveRegion string
+}
 
 func NewCmdApprove() *cobra.Command {
+	o := &ApproveOptions{}
 	cmd := &cobra.Command{
-		Use:   "approve",
-		Short: "approve enrollment-request fleet-name",
+		Use:   "approve enrollmentrequest/NAME",
+		Short: "Approve an enrollment request.",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return RunApprove(cmd.Context(), args[0])
+			if err := o.Complete(cmd, args); err != nil {
+				return err
+			}
+			if err := o.Validate(args); err != nil {
+				return err
+			}
+			return o.Run(cmd.Context(), args)
 		},
 		SilenceUsage: true,
 	}
 
-	cmd.Flags().StringArrayVarP(&approveLabels, "label", "l", []string{}, "labels to add to the device")
-	cmd.Flags().StringVarP(&approveRegion, "region", "r", "default", "region for the device")
+	cmd.Flags().StringArrayVarP(&o.ApproveLabels, "label", "l", []string{}, "Labels to add to the device, as a comma-separated list of key=value.")
```

Comment:
while Use and Short is ok to start with capital letters, I think the standard for flags is lower case. Not strict rule though.

Reply:
Do you a pointer to best-practices or style guides? I'm ok either way, but this follows what kubectl standardises on.